cmd/cloud/deployment.go

```diff
@@ -864,8 +866,22 @@ func deploymentOverrideHibernation(cmd *cobra.Command, args []string, isHibernat
 	return deployment.UpdateDeploymentHibernationOverride(deploymentID, ws, deploymentName, isHibernating, overrideUntil, forceOverride, platformCoreClient)
 }
 
-func isValidExecutor(executor string) bool {
-	return strings.EqualFold(executor, deployment.KubeExecutor) || strings.EqualFold(executor, deployment.CeleryExecutor) || executor == "" || strings.EqualFold(executor, deployment.CELERY) || strings.EqualFold(executor, deployment.KUBERNETES)
+func isValidExecutor(executor, runtimeVersion string) bool {
```

Comment:
I think we have the exact same method declared twice in two separate packages and could be moved into a single place.

Reply:
Yeah it was already that way, but I took this opportunity to consolidate them, thanks for the note.